src/email-templates/rejection_notification.handlebars

```diff
@@ -0,0 +1,8 @@
+<p>Hello {{companyName}},</p>
+<br>  
+<p>We have reviewed your application for {{companyName}} and decided to reject your application.</p>    
+<p>Some common causes of rejection are spam, abuse and innapropriate motivation. 
```

Comment:
`s/and/or`

Reply:
innapropriate sounds weird as well, we might want to clarify with something like "or a motivation that does not align with the platform's principles" or "our principles". Not sure though, the big boss @andrefmrocha should call this one 😁

Reply:
"or a motivation that does not align with our principles" sounds great in this case.

Reply:
Yep, this was a mere placeholder for review. I didn't really know what to put there... Changing